clients/java/src/main/java/io/camunda/client/impl/oauth/OAuthCredentialsProviderBuilder.java

```diff
@@ -269,6 +278,34 @@ public Duration getReadTimeout() {
     return readTimeout;
   }
 
+  public OAuthCredentialsProviderBuilder entraCertificatePath(final String entraCertificatePath) {
+    if (entraCertificatePath != null) {
+      this.entraCertificatePath = Paths.get(entraCertificatePath);
+    }
+    return this;
+  }
+
+  public Path getEntraCertificatePath() {
+    return entraCertificatePath;
+  }
+
+  public OAuthCredentialsProviderBuilder entraCertificatePassword(
+      final String entraCertificatePassword) {
+    this.entraCertificatePassword = entraCertificatePassword;
+    return this;
+  }
+
+  public String getEntraCertificatePassword() {
+    return entraCertificatePassword;
+  }
+
+  public boolean entraConfigurationProvided() {
```

Comment:
🔧 I'd remove the Entra part of this, although our implementation right now is based on a requirement when using MS Entra, it may not be the only platform that requires it